pypesto/objective/amici_objective.py

```diff
@@ -200,8 +209,7 @@ def __deepcopy__(self, memodict=None):
         edatas = [amici.ExpData(data) for data in self.edatas]
         other = AmiciObjective(model, solver, edatas)
         for attr in self.__dict__:
-            if attr not in ['amici_solver', 'amici_model',
-                            'edatas', 'preequilibration_edatas']:
+            if attr not in ['amici_solver', 'amici_model', 'edatas']:
```

Comment:
I guess the edatas in preeq_guesses will cause problems just as the former preequilibration_edatas?

Reply:
no, the guesses are regular dicts of numpy arrays, so regular deepcopy will just work fine on them.